src/prefect/_internal/pydantic/v2_schema.py

```diff
@@ -74,14 +68,6 @@ def process_v2_params(
 
     type_ = t.Any if param.annotation is inspect.Parameter.empty else param.annotation
 
-    # Replace pendulum type annotations with our own so that they are pydantic compatible
-    if type_ == pendulum.DateTime:
-        type_ = PydanticPendulumDateTimeType
-    if type_ == pendulum.Date:
-        type_ = PydanticPendulumDateType
-    if type_ == pendulum.Duration:
-        type_ = PydanticPendulumDurationType
-
```

Comment:
it's possible that someone may encounter a difference in behavior here, though no tests failed as a result of this change